en/ui-system/components/editor/ui-opacity.md

```diff
@@ -2,6 +2,19 @@
 
 The UIOpacity Component records a transparency modification flag for the node, which is used to influence all the render nodes inside its sub tree. Normally it's used on a non render nodes, otherwise its opacity will be multiplied with the render component's opacity. The render nodes can set transparency individually by setting the alpha channel of `color`.
 
+The method of use is as follows
+
+![ui-opacity](uiopacity/ui-opacity.png)
+
+You can also set transparency by code
```

Comment:
You can also set transparency by code
-->
You can also set transparency by code. Example: